SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -746,9 +745,9 @@ NodeProxy : BusPlug {
 
 	// bundle: apply the node map settings to the entire group
 	sendAllToBundle { | bundle, extraArgs |
-		extraArgs = nodeMap.asOSCArgArray ++ extraArgs.value.asOSCArgArray;
+		var args = Thunk { nodeMap.asOSCArgArray ++ extraArgs.value.asOSCArgArray };
```

Comment:
better style IMO would be:

```supercollider
Thunk({ ... })
```
which makes it clear that you are creating a Thunk object.